lib/constants.go

```diff
@@ -1822,7 +1822,11 @@ const (
 	// key represents who to pay the fee to and the value indicates how many basis points to charge. All fees are
 	// charged as "taker fees" and are paid to the pkid in the map in the *quote currency* of the market (typically
 	// DESO, USDC, or FOCUS).
-	TokenTradingFeesByPkidMapKey = "TokenTradingFeesByPkidMap"
+	TokenTradingFeesByPkidMapKey            = "TokenTradingFeesByPkidMap"
+	DisableTradingFeeUpdateKey              = "DisableTradingFeeUpdateKey"
+	CreatorRevsharePercentageBasisPointsKey = "CreatorRevsharePercentageBasisPoints"
+	DisableCreatorRevshareUpdateKey         = "DisableCreatorRevshareUpdateKey"
+	CoinCategoryExtraDataKey                = "CoinCategoryExtraDataKey"
```

Comment:
Any way we can shorten these strings? It's just a lot of extra space in each transaction you'll be using.

Reply:
It's fine here, but in the future as you define new ones, please start with shortened strings. We should force folks to pay for more than they need to when transacting.